integration/apply_test.go

```diff
@@ -59,6 +59,17 @@ func TestApply(t *testing.T) {
 		assert.NoError(t, err, "Failed to apply resource")
 	})
 
+	t.Run("snyk", func(t *testing.T) {
+		// Create a new server route for mocking a Bubbly server response
+		bCtx := env.NewBubblyContext()
+		bCtx.UpdateLogLevel(zerolog.DebugLevel)
+		err := bubbly.Apply(bCtx, "./testdata/snyk")
+		assert.NoError(t, err, "Failed to apply resource")
+
+		// test that `bubbly get all` returns valid resources from the apply
+		testGet(t, bCtx, []string{"all"})
```

Comment:
```suggestion
		testGet(t, bCtx, []string{"extract/snyk"})
```

might be more valuable?

Reply:
Good call!